packages/theme-default/src/layout.tsx

```diff
@@ -27,7 +27,13 @@ const Features = features => (
   <div className="__dumi-default-layout-features">
     {features.map(feat => (
       <dl key={feat.title} style={{ backgroundImage: feat.icon ? `url(${feat.icon})` : undefined }}>
-        <dt>{feat.title}</dt>
+        {
+          feat.link
+            ? <Link to={feat.link} key={feat.title}>
```

Comment:
上面是因为有循环，还是去掉了，过段时间可能会以为是为了特殊处理什么 bug 才加的😄
```suggestion
            ? <Link to={feat.link}>
```